UPGRADING.md

```diff
@@ -0,0 +1,5 @@
+# Upgrading
+
+## 0.3.7 to 1.0.0
```

Comment:
Change to `## Upgrading to >= 1.0.0` and add a subsection below `### Mismatched Sort Fields` or something like that. Don't make the text a bullet. This way we can have nice multi-line upgrade notes.